xml/cap_troubleshooting.xml

```diff
@@ -452,14 +452,17 @@ end
   </sect2>
  </sect1>
  <sect1 xml:id="sec-cap-tbl-mysql-failure">
-  <title><literal>mysql</literal> Pods Fail to Start</title>
+  <title>Pods Fail to Start</title>
```

Comment:
We should probably indicate that this section if for mysql pods only since we currently have a section for the api-group pod issue and the autoscaler pod issue.

Removing mysql from the title would make more sense if we're planning to consolidate the api-group and autoscaler examples as subsections under this section.